app/main.py

```diff
@@ -15,11 +15,15 @@
 from contextlib import asynccontextmanager
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
 from apscheduler.triggers.cron import CronTrigger
-
 import os
 import logging
+import fcntl
+import tempfile
 from datetime import UTC
 
+# Set timezone environment variable to prevent tzlocal warning
+os.environ['TZ'] = 'UTC'
```

Comment:
nit: guessing it's fine but this feels a bit weird (I'm not familiar with this error so don't have an alternate proposal).